includes/admin/models/class-wppr-review-model.php

```diff
@@ -884,6 +894,20 @@ public function set_image( $image ) {
 		return false;
 	}
 
+	/**
+	 * Return the review image ID.
+	 *
+	 * @since   3.4.3
+	 * @access  public
+	 * @return int
+	 */
+	public function get_wppr_image_id() {
+		global $wpdb;
+		$attachment  = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM $wpdb->posts WHERE guid=%s", $this->image ) );
```

Comment:
@UVLabs hiding SQL errors would be a good option https://codex.wordpress.org/Class_Reference/wpdb#Show_and_Hide_SQL_Errors

Reply:
@contactashish13 it seems errors are suppressed/hidden by default as noted in the descriptions here:

https://developer.wordpress.org/reference/classes/wpdb/hide_errors/

https://developer.wordpress.org/reference/classes/wpdb/suppress_errors/

Reply:
I can confirm that errors are hidden by default, I added a faulty query and no errors showed, then I added $wpdb->show_errors(); and then the error showed

Reply:
same thing, would be better to name it get_image_id.